### PR TITLE
Edits to issue templates

### PR DESCRIPTION
**Pull request by abbycross**
From @javabster -- improvements to issue templates:

### content bug or inaccuracy

- [x] can we add a checkbox selection like we had in the old one? something like this:

```
- [ ] typo
- [ ] code bug
- [ ] out-of-date content
- [ ] it's just wrong
- [ ] other
```

- [x] description formatting is a bit weird with the slash (/). Can you update to say "Report typos, bugs, out-of-date content, broken links etc."

- [x] can you have it automatically add the "bug" label when the issue is opened?

### UI issue

- [x] typo in the "please describe the UI problem" description, there is an extra ) at the end

- [x] also can you have it automatically add a label, might need to make a new one called "UI feedback" or something

### Request new content
- [x] could you rephrase the last question in the form to emphasise that it is optional? i imagine some people might just want to add a suggestion but dont want to be further involved at all, they just want to submit a request and leave. That currently doesn't have an option in the dropdown and the q is mandatory. You could rephrase the question to "if this new content request is accepted, do you want to write the content?" and then have all the dropdown options including a "no" option
- [x]  can you automatically add the "new" label and also a "content request" label (i think the second will need to be created
### API refs
add the links to the 1 line description of the issue
- [x] https://docs.quantum-computing.ibm.com/api/qiskit
- [x] https://docs.quantum-computing.ibm.com/api/qiskit-ibm-runtime/runtime_service

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/new-content.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 name: ✨ Request new content
 description: Make a request to the Content Team to add a new section in the docs.
-labels: ["NEW CONTENT REQUEST ✨"]
+labels: ["content request", "new"]
```
Comment (javabster) [inline .github/ISSUE_TEMPLATE/new-content.yaml:3]:
```suggestion
labels: ["content request", "new ✨"]
```

**Review by javabster — APPROVED** [review]
LGTM 🚀 

we should check that the automatic labelling action is working properly once this merges

File: .github/ISSUE_TEMPLATE/new-content.yaml (hunk 1 above)
Comment (abbycross) [reply .github/ISSUE_TEMPLATE/new-content.yaml:3]: Aha, you've already done so

**Comment by abbycross** [comment]
Confirmed that the automatic labels are working now 🥳
